extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/shared.properties.js

```diff
@@ -31,16 +29,14 @@ angular.module('PaperUI.controllers').service('sharedProperties', function() {
             var modArr = this.getModuleArray(type);
 
             if (!value.id) {
-                value.id = type + "_" + tId;
+                value.id = type.substring(0, 1) + "_" + Date.now();
```

Comment:
Will this now add a timestamp as part of the id? I would still prefer "simple" ids, e.g. using the first positive integer that isn't yet used within the rule. Would something like that be possible?